py-polars/polars/dataframe/frame.py

```diff
@@ -6823,10 +6823,6 @@ def join(
             Note that joining on any other expressions than `col`
             will turn off coalescing.
 
-        Returns
-        -------
-        DataFrame
-
```

Comment:
Drive by: Returns section is not consistent with other DataFrame methods and `LazyFrame.join` docstring. Is also already covered by type hint.